romancal/resample/meta_blender.py

```diff
@@ -80,9 +81,31 @@ def __init__(self):
 
     def _blend_first(self, model):
         # make a blank mosic metdata node
-        self._model = maker_utils.mk_datamodel(
-            datamodels.MosaicModel, n_images=0, shape=(0, 0)
+        # FIXME includes fake values to match the previous maker_utils
```

Comment:
Several of the fake values here (pixel area -999999, etc) are reproducing current bugs on main. A follow-up PR can drop these fake values and replace them with real data. xref: https://github.com/spacetelescope/romancal/issues/1681